.github/workflows/pr.yaml

```diff
@@ -0,0 +1,56 @@
+name: "Integration Tests"
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  integration_tests:
+    runs-on: ubuntu-latest
```

Comment:
Could we use latest `ubuntu-20.04` instead?

Reply:
Done